{{cookiecutter.app_name}}/Makefile

```
@@ -13,7 +13,11 @@ env-teardown:
 
 db/migrate:
 	make wait-for-postgres
-	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" up
+ifneq ("$(wildcard $(database/migrations/.keep))","")
+  goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" up
+else
+	$(info NO migration files to run)
```

```suggestion
ifeq ("$(wildcard $(database/migrations/.keep))","")
  $(info NO migration files to run)
else
	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" up
```

Not sure if that work, but can we use an `ifeq` so we don't need to process the negative form (if NOT X do A else B `vs` if X do B else A`)

---

It works so changed in 1183eb4 🙌🏼